shielder/contract/mocked_zk/mod.rs

```diff
@@ -1,9 +1,11 @@
-mod account;
-mod note;
-mod ops;
+pub mod account;
+#[cfg(feature = "std")]
```

Comment:
using a `Wrapping` utility shouldn't be a sufficient reason to hide whole, basic module here

Reply:
moved to `test_utils` module, this merkle tree is only for tests